### ISSUE TITLE
--fqdn seems required but is not showing up like that in help

### ISSUE THREAD

**Issue by kargakis** [Azure/OpenShift#18]
```console
# az openshift create -g kargakisprod -n kargakisprod -l westeurope
Invalid value specified for property 'displayName' of resource 'Application'.
bash-4.4# az openshift create --help

Command
    az openshift create : Create a new managed OpenShift cluster.

Arguments
    --name -n           [Required] : Name of the managed OpenShift cluster.
    --resource-group -g [Required] : Name of resource group. You can configure the default group
                                     using `az configure --defaults group=<name>`.
    --aad-client-app-id            : The ID of an Azure Active Directory client application of type
                                     "Native".
    --aad-client-app-secret        : The secret of an Azure Active Directory client application.
    --aad-tenant-id                : The ID of an Azure Active Directory tenant.
    --fqdn                         : FQDN for OpenShift API server loadbalancer internal hostname.
                                     For example myopenshiftcluster.eastus.cloudapp.azure.com.
bash-4.4# az openshift create -g kargakisprod -n kargakisprod -l westeurope --fqdn kargakisprod.westeurope.cloudapp.azure.com
# Failed with a different error
```

**Comment by julienstroheker** [comment]
This has been updated. Thanks @kargakis 

```
az openshift create --help

Command
    az openshift create : (PREVIEW) Create a new managed OpenShift cluster.

Arguments
    --fqdn       [Required] : FQDN for OpenShift API server loadbalancer internal hostname. For
                              example myopenshiftcluster.eastus.cloudapp.azure.com.
    --name -n    [Required] : Name of the managed OpenShift cluster.
    --aad-client-app-id     : The ID of an Azure Active Directory client application. If not
                              specified, a new Azure Active Directory client is created.
    --aad-client-app-secret : The secret of an Azure Active Directory client application.
    --aad-tenant-id         : The ID of an Azure Active Directory tenant.
    --location -l           : Location. You can configure the default location using `az configure
                              --defaults location=<location>`.  Default: eastus.
    --no-wait               : Do not wait for the long-running operation to finish.
    --node-count -c         : Number of nodes in the OpenShift node pool.  Default: 4.
    --node-vm-size -s       : Size of Virtual Machines to create as OpenShift nodes.  Default:
                              Standard_D4s_v3.
    --resource-group -g     : Name of resource group. You can configure the default group using `az
                              configure --defaults group=<name>`.  Default: myResourceGroup.
    --subnet-cidr           : The CIDR used on the Subnet into which to deploy the cluster.
                              Default: 10.0.0.0/24.
    --tags                  : Space-separated tags in 'key[=value]' format. Use '' to clear existing
                              tags.
    --vnet-cidr             : The CIDR used on the VNet into which to deploy the cluster.  Default:
                              10.0.0.0/8.
    --vnet-peer-id          : The ID of a subnet in an existing VNet into which to peer the cluster.

Global Arguments
    --debug                 : Increase logging verbosity to show all debug logs.
    --help -h               : Show this help message and exit.
    --output -o             : Output format.  Allowed values: json, jsonc, table, tsv, yaml.
                              Default: json.
    --query                 : JMESPath query string. See http://jmespath.org/ for more information
                              and examples.
    --subscription          : Name or ID of subscription. You can configure the default subscription
                              using `az account set -s NAME_OR_ID`.
    --verbose               : Increase logging verbosity. Use --debug for full debug logs.

Examples
    Create an OpenShift cluster and auto create an AAD Client
        az openshift create -g MyResourceGroup -n MyManagedCluster --fqdn <FQDN>

    Create an OpenShift cluster with 5 compute nodes and a custom AAD Client.
        az openshift create -g MyResourceGroup -n MyManagedCluster --fqdn <FQDN> --aad-client-app-id
        <APP_ID> --aad-client-app-secret <APP_SECRET> --aad-tenant-id <TENANT_ID> --node-count 5

    Create an Openshift cluster using a custom vnet
        az openshift create -g MyResourceGroup -n MyManagedCluster --fqdn <FQDN> --vnet-peer-id
        "/subscriptions/0000000-0000-0000-0000-000000000000/resourceGroups/openshift-
        vnet/providers/Microsoft.Network/virtualNetworks/test"
```

**Comment by charlesakalugwu** [comment]
@julienstroheker and now `Resource Group` is optional?

I also saw the same issues as @kargakis earlier in the day but now I am trying again with `fqdn` specified and so far I am not seeing any errors. I'll update this later with the results of my tests.

```
bash-4.4# az openshift create --name charlesde --resource-group charlesde --location westeurope
Invalid value specified for property 'displayName' of resource 'Application'.

bash-4.4# az openshift create --name charlesde --resource-group charlesde --location westeurope --fqdn charlesde.westeurope.cloudapp.azure.com
 - Running ..
```

**Comment by charlesakalugwu** [comment]
@kargakis  I was  able to 

- create a default cluster
- scale out the cluster agents from 4 to 5 nodes
- scale in the cluster agents from 5 to 3 nodes
- get details of the cluster along the previous steps
- delete the cluster

All in `westeurope`

**Comment by julienstroheker** [comment]
Hi @charlesakalugwu 

Thanks for your tests. 

About the RG, the scope is global, the CLI can handle a "default" RG value for all the commands. 
When I printed earlier the command, I had a value already set for my resource group so the CLI did not show it as `required`.

TLDR: `name` `resource-group` and `fqdn` are mandatory.

Hope it makes sense.

**Comment by charlesakalugwu** [comment]
@julienstroheker makes sense. thanks!
